compose/src/commonMain/kotlin/com/boswelja/ephemeris/compose/EphemerisCalendar.kt

```diff
@@ -24,23 +23,26 @@ public fun EphemerisCalendar(
     dayContent: @Composable BoxScope.(DisplayDate) -> Unit
 ) {
     val pagerState = rememberInfinitePagerState()
-    LaunchedEffect(pagerState) {
-        snapshotFlow { pagerState.page }.collect {
-            calendarState.currentMonth = calendarState.calendarPageLoader.monthFor(it.toLong())
-        }
+    val coroutineScope = rememberCoroutineScope()
+//    LaunchedEffect(pagerState) {
```

Comment:
👀

Reply:
currentMonth doesn't exist in the new CalendarState spec, I just haven't implemented the alternative yet